config/resources/submissions-domain.lisp

```diff
@@ -0,0 +1,110 @@
+(define-resource submission ()
+  :class (s-prefix "subm:Indiening")
+  :properties `((:short-title         :string ,(s-prefix "dct:alternative"))
+                (:title               :string ,(s-prefix "dct:title"))
+                (:confidential        :boolean   ,(s-prefix "ext:vertrouwelijk"))
+                (:subcase-name        :string ,(s-prefix "ext:procedurestapNaam"))
+                (:planned-start       :datetime ,(s-prefix "subm:geplandeStart"))
+                (:created             :datetime ,(s-prefix "dct:created"))
+                (:modified            :datetime ,(s-prefix "ext:modified"))
+                (:approved-by         :uri-set ,(s-prefix "subm:goedkeurder"))
+                (:approval-comment    :string ,(s-prefix "subm:goedkeuringOpmerking"))
+                (:notified            :uri-set ,(s-prefix "subm:genotificeerde"))
+                (:notification-comment :string ,(s-prefix "subm:ccOpmerking")))
+  :has-one `((decisionmaking-flow     :via ,(s-prefix "subm:ingediendVoor")
+                                      :as "decisionmaking-flow")
+             (subcase                 :via ,(s-prefix "subm:ingediendVoorProcedurestap")
+                                      :as "subcase")
+             (subcase-type            :via ,(s-prefix "dct:type")
+                                      :as "type")
+             (concept                 :via ,(s-prefix "ext:agendapuntType")
+                                      :as "agenda-item-type")
+             (concept                 :via ,(s-prefix "adms:status")
+                                      :as "status")
```

Comment:
add comment: this is the submissions status (concept-scheme `thConceptScheme:ebfe253c-0537-11ee-bb35-ee395168dcf7`)